UI/MVUX/src/MVUX/Presentation/FeedSample/FeedModel.cs

```diff
@@ -0,0 +1,26 @@
+using Uno.Extensions.Reactive;
+
+namespace MVUX.Presentation.FeedSample;
+
+public partial record FeedModel
+{
+	public IFeed<Person> Person { get; }
+	
+	public FeedModel()
+	{
+		Person = Feed.Async(async ct =>
+		{
+			await Task.Delay(2000, ct);
+			
+			var people = new[]
+			{
+				new Person("Master", "Yoda"),
+				new Person("Darth", "Vader"),
+				new Person("Luke", "Skywalker")
+			};
+			
+			return people[new Random().Next(people.Length)];
+		});
```

Comment:
Please use getter syntax. Feed syntax was designed to avoid such noise. Also considering the comment in the XAML, I would move the `people` list init out of the feed declaration: 
```suggestion
	private static readonly Person[] _people = new[]
			{
				new Person("Master", "Yoda"),
				new Person("Darth", "Vader"),
				new Person("Luke", "Skywalker")
			};

	public IFeed<Person> Person => Feed.Async(async ct =>
		{
			await Task.Delay(2000, ct); // Simulate network delay
						
			return _people[Random.Shared.Next(people.Length)];
		});
```